contrib/endpoints/src/api_manager/check_security_rules.cc

```diff
@@ -14,92 +14,339 @@
 //
 ////////////////////////////////////////////////////////////////////////////////
 #include "contrib/endpoints/src/api_manager/check_security_rules.h"
+#include "contrib/endpoints/src/api_manager/auth/lib/json_util.h"
+#include <sstream>
+#include <iostream>
 
-#include <string>
-
-#include "contrib/endpoints/include/api_manager/api_manager.h"
-#include "contrib/endpoints/include/api_manager/request.h"
-
+using ::google::api_manager::auth::GetProperty;
+using ::google::api_manager::auth::GetStringValue;
 using ::google::api_manager::utils::Status;
+using ::google::protobuf::util::error::Code;
 
 namespace google {
 namespace api_manager {
 
-namespace {
-
 const char kFirebaseServerStaging[] =
     "https://staging-firebaserules.sandbox.googleapis.com/";
 
-// An AuthzChecker object is created for every incoming request. It does
-// authorizaiton by calling Firebase Rules service.
```

Comment:
Added it back. Initially I had code that was creating the object in the service context but this did not work due to circular dependences. I removed it as a part of that change and forgot to add it back.